### DISCUSSION TITLE
Lightning Hydra vs LightningCLI

### DISCUSSION THREAD

**Discussion by rabinadk1** [ashleve/lightning-hydra-template#627] [Q&A]
What would be critical differences in the workflow when Lightning CLI is used instead of lightning hydra? What would be the advantages and disadvantages of using one over another?

**Comment by tshu-w** [comment]
FYI, I'm happy with Lightning CLI, here is the [lightning-template](https://github.com/tshu-w/lightning-template) I maintain to align lightning-hydra-template.
The only missing is the sweep feature of hydra, I implement a [sweep cli](https://github.com/tshu-w/lightning-template/blob/main/src/utils/sweep_cli.py) through `ray[tune]` but it is difficult to cover all situations.
